doc/cookbook/source/examples/evaluation/cross_validation.rst

```diff
@@ -0,0 +1,101 @@
+============================================
+Cross Validation
+============================================
+
+Cross Validation is a type of model validation technique for figuring out how the
+results from a statistical analysis will generalize to an independent data set. It
+essentially involves training the model on a subset of the data, and testing on the
+remainder, then repeating this process for several complementary partitions and then
+combining all the validation results generated (For instance, an average) to estimate
+a performance for the final predictive model.
```

Comment:
remove "a performance for the final predictive model"

Reply:
Done